### PR TITLE
Feat: Add /logout route, redirect there from logout button

### PR DESCRIPTION
**Pull request by realmayus**
Add /logout route, redirect there from logout button

Closes https://github.com/corona-school/project-user/issues/1234

### CODE REVIEW

File: src/components/Logout.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import useApollo from '../hooks/useApollo';
+import { useNavigate } from 'react-router-dom';
+
+export default function Logout() {
+    const navigate = useNavigate();
+    const { logout } = useApollo();
+    logout();
```
Comment (Jonasdoubleyou) [inline src/components/Logout.tsx:7]: It's a Promise, better wait for it :)

File: src/components/Logout.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            } catch (error) {
+                logError('Authentication', 'Failed to logout', error);
+            }
+        }, []);
+    };
+
+    const logout = useLogout();
+    logout().then(() => navigate('/welcome', { state: locState }));
+
+    return null;
```
Comment (Jonasdoubleyou) [inline src/components/Logout.tsx:35]: Maybe a loading spinner instead of an empty page?

File: src/components/Logout.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            try {
+                await logout();
+            } catch (error) {
+                logError('Authentication', 'Failed to logout', error);
+            }
+        }, []);
+    };
+
+    const logout = useLogout();
+    logout().then(() => navigate('/welcome', { state: locState }));
```
Comment (Jonasdoubleyou) [inline src/components/Logout.tsx:33]: If we login again, on which page will we land?

Comment (realmayus) [reply src/components/Logout.tsx:33]: We land on /start

**Review by Jonasdoubleyou — APPROVED** [review]

**Pull request merged**